IfElseSubstitution not in Humble

Hi! :wave: 

I've run into the issue with the **humble** branch where it looks like there is a use of `IfElseSubstitution` that doesn't exist. I can see from [here](https://github.com/ros2/launch/blob/rolling/launch/launch/substitutions/if_else_substitution.py) that it is defined in rolling but not humble. I'm on Ubuntu 22.04.4 LTS.

For reference:
```
[ERROR] [launch]: Caught exception in launch (see debug for traceback): Caught multiple exceptions when trying to load file of format [py]:
 - ImportError: cannot import name 'IfElseSubstitution' from 'launch.substitutions' (/opt/ros/humble/lib/python3.10/site-packages/launch/substitutions/__init__.py)
 - InvalidFrontendLaunchFileError: The launch file may have a syntax error, or its format is unknown
```
I managed to resolve it by using `IfCondition` and `UnlessCondition`. I've opened a [PR](#44) with my fix for review. I'd appreciate it if someone could take a look when they have a chance. Thanks!

--- a/ur_simulation_gz/launch/ur_sim_control.launch.py
+++ b/ur_simulation_gz/launch/ur_sim_control.launch.py
@@ -43,7 +43,6 @@ from launch.substitutions import (
     FindExecutable,
     LaunchConfiguration,
     PathJoinSubstitution,
-    IfElseSubstitution,
 )
 from launch_ros.actions import Node
 from launch_ros.substitutions import FindPackageShare
@@ -168,18 +167,24 @@ def launch_setup(context, *args, **kwargs):
             "true",
         ],
     )
-
-    gz_launch_description = IncludeLaunchDescription(
+    gz_launch_description_with_gui = IncludeLaunchDescription(
         PythonLaunchDescriptionSource(
             [FindPackageShare("ros_gz_sim"), "/launch/gz_sim.launch.py"]
         ),
         launch_arguments={
-            "gz_args": IfElseSubstitution(
-                gazebo_gui,
-                if_value=[" -r -v 4 ", world_file],
-                else_value=[" -s -r -v 4 ", world_file],
-            )
+            "gz_args": ["-r", "-v", "4", world_file]
         }.items(),
+        condition=IfCondition(gazebo_gui)
+    )
+
+    gz_launch_description_without_gui = IncludeLaunchDescription(
+        PythonLaunchDescriptionSource(
+            [FindPackageShare("ros_gz_sim"), "/launch/gz_sim.launch.py"]
+        ),
+        launch_arguments={
+            "gz_args": ["-s", "-r", "-v", "4", world_file]
+        }.items(),
+        condition=UnlessCondition(gazebo_gui)
     )
 
     nodes_to_start = [
@@ -189,7 +194,8 @@ def launch_setup(context, *args, **kwargs):
         initial_joint_controller_spawner_stopped,
         initial_joint_controller_spawner_started,
         gz_spawn_entity,
-        gz_launch_description,
+        gz_launch_description_with_gui,
+        gz_launch_description_without_gui,
     ]
 
     return nodes_to_start

